src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -280,7 +280,9 @@ class RemoteMachineTrainingService implements TrainingService {
             const jobpidPath: string = this.getJobPidPath(trialJob.id);
             try {
                 await SSHClientUtility.remoteExeCommand(`pkill -P \`cat ${jobpidPath}\``, sshClient);
-                trialJob.status = getJobCancelStatus(isEarlyStopped);
+                //TODO: delete and move set USER_CANCELLED/EARLY_STOP in getTrialJob
+                // Mark the toEarlyStop tag here
+                trialJob.isEarlyStopped = isEarlyStopped;
```

Comment:
this line should be put before the `await SSHClientUtility` (i.e., line 282)

Reply:
Done